core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -63,8 +63,7 @@
  */
 public final class CoreSocketFactory {
   public static final String CLOUD_SQL_INSTANCE_PROPERTY = "cloudSqlInstance";
-  public static final String MYSQL_SOCKET_FILE_FORMAT = "/cloudsql/%s";
-  public static final String POSTGRES_SOCKET_FILE_FORMAT = "/cloudsql/%s/.s.PGSQL.5432";
+  public static final String UNIX_SOCKET_PROPERTY = "unixSocketPath";
```

Comment:
Does this have to be public? I don't think it's used anywhere except in this file.